### ISSUE TITLE
Error Uploading Videos with disabled thumbnails

### ISSUE THREAD

**Issue by Feilner** [flingo64/PhotoStation-Upload-Lr-Plugin#62]
I have disabled the generation of Thumbnails.

When I try to upload videos I got this error. Which is irritating me.
```
Photo StatLr: That does it, I'm leaving!
Ein interner Fehler ist aufgetreten: [string
"PSUploadTask.lua"1:352:attempttoindexlocal
'thumbSettings' (a boolean value)
```

**Comment by flingo64** [comment]
Seems to be the same as issue #60, but for videos.
I'm on vacation, so i cannot fix within the next weeks, but you may try the same fix as for #60: in line 352 replace all occurences of 
`thumbSettings.XXX_Filename`
by 
`thumbSettings and thumbSettings.XXX_Filename`
where XXX is in XL, L, M, B, S

**Comment by flingo64** [comment]
Fixed in v7.2.3. Please re-test.
